src/Common/FunctionDocumentation.h

```diff
@@ -48,12 +48,19 @@ struct FunctionDocumentation
 
     struct Argument
     {
-        std::string name;
-        std::string description;
+        std::string name;                     /// E.g. "y"
+        std::string description;              /// E.g. "The divisor."
+        std::vector<std::string> types = {};  /// E.g. {"(U)Int*", "Float*"}
+                                              /// Default initialized only during a transition period, see 'argumentsAsString'.
     };
     using Arguments = std::vector<Argument>;
 
-    using ReturnedValue = std::string;
+    struct ReturnedValue
+    {
+        std::string description;              /// E.g. "Returns the remainder of the division."
+        std::vector<std::string> types = {};  /// E.g. {"Float32"}
```

Comment:
I can't think right now of functions with different return types but perhaps such functions exist.
Anyways, we can keep the new `types` argument as vector instead of string just for the sake of consistency with l. 53.